src/cryptonote_protocol/cryptonote_protocol_defs.h

```diff
@@ -69,6 +69,8 @@ namespace cryptonote
 	
 	uint64_t avg_upload;
 	uint64_t current_upload;
+  
+    uint32_t support_flags;
```

Comment:
I feel pedantic, but... can you keep the same indent as the previous fields please :)